src/libs/SearchUtils.ts

```diff
@@ -298,5 +299,21 @@ function getSearchParams() {
     return topmostCentralPaneRoute?.params as AuthScreensParamList['Search_Central_Pane'];
 }
 
-export {getListItem, getQueryHash, getSections, getSortedSections, getShouldShowMerchant, getSearchType, getSearchParams, shouldShowYear, isReportListItemType, isTransactionListItemType};
+function isSearchResultEmpty(searchResults: SearchResults) {
```

Comment:
how about `isSearchResultsEmpty`? I know its bad english, but everywhere else we call this data `SearchResultS`

Reply:
Suggesting that we may also use `doesContainTransactions`, since that is what we check for here. /Nab

Reply:
Whoops, I've marked it as resolved and didn't notice your comment. I think both names are fine, if needed I can change it.